src/backend/distributed/commands/multi_copy.c

```diff
@@ -631,7 +651,10 @@ CopyToNewShards(CopyStmt *copyStatement, char *completionTag, Oid relationId)
 										copyStatement->options);
 
 	CopyOutState copyOutState = (CopyOutState) palloc0(sizeof(CopyOutStateData));
-	copyOutState->binary = true;
+	copyOutState->delim = delimiterCharacter;
+	copyOutState->null_print = nullPrintCharacter;
+	copyOutState->null_print_client = nullPrintCharacter;
```

Comment:
I found the problem, somehow I added a line something like;
`make -s` 
to build script which I created for my local use. I removed -s parameter and it shows all warnings now.